setup/pkg/fieldlabs/environment_create.go

```diff
@@ -313,6 +341,18 @@ func (e *EnvironmentManager) createVendorLabs(envs []Environment, labSpecs []Lab
 				return nil, errors.Wrapf(err, "promote release %d to channel %q", release.Sequence, channel.Slug)
 			}
 
+			for _, extraRelease := range lab.Status.ExtraReleases {
+				releaseInfo, err := e.Client.CreateRelease(app.ID, extraRelease.YAML)
+				if err != nil {
+					return nil, errors.Wrapf(err, "create release for %q", extraRelease.Spec.YAMLDir)
+				}
+				extraRelease.Release = releaseInfo
+
+				if extraRelease.Spec.PromoteChannel != "" {
+					return nil, errors.Errorf("promoting extra releases not implemented, bailing on %q", extraRelease.Spec.YAMLDir)
```

Comment:
Maybe `continue` with some message to the console it ain't supported yet? Now it will just stop the app and release creation process, correct?